pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cognite-neat"
-version = "0.111.0"
+version = "0.0.0"
```

Comment:
Can you explain we reset the version to 0.0.0? Isn't the package already published somewhere and we will have conflicts?

Reply:
The `0.0.0` is a placeholder. The CD process replaces it before release with the actual version. 
![image](https://github.com/user-attachments/assets/6d190529-8a03-4972-a477-6ed4de610a9d)